### PR TITLE
Add docker image + update packages

### PR DESCRIPTION
**Pull request by xepozz**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
+FROM node:21.7-alpine as build
+
+RUN apk add --update git nano curl python3 py3-setuptools make g++\
```
Comment (samdark) [inline Dockerfile:3]:
```suggestion
RUN apk add --update --no-cache git nano curl python3 py3-setuptools make g++\
```

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+COPY lerna.json package*.json ./
+
+# Dev Panel
+COPY packages/yii-dev-panel/package*.json packages/yii-dev-panel/
+
+# Dev Panel SDK
+COPY packages/yii-dev-panel-sdk/package*.json packages/yii-dev-panel-sdk/
+
+# Dev Toolbar
+COPY packages/yii-dev-toolbar/package*.json packages/yii-dev-toolbar/
```
Comment (samdark) [inline Dockerfile:18]: What changes more often?

**Review by samdark — APPROVED** [review]

File: Dockerfile (hunk 2 above)
Comment (xepozz) [reply Dockerfile:18]: I think yii-dev-panel itself + sdk

Comment (samdark) [reply Dockerfile:18]:
```suggestion
# Dev Toolbar
COPY packages/yii-dev-toolbar/package*.json packages/yii-dev-toolbar/

# Dev Panel
COPY packages/yii-dev-panel/package*.json packages/yii-dev-panel/

# Dev Panel SDK
COPY packages/yii-dev-panel-sdk/package*.json packages/yii-dev-panel-sdk/
```

Comment (samdark) [reply Dockerfile:18]: Reason: layer with toolbar won't be rebuilt every time other layers change.

Comment (xepozz) [reply Dockerfile:18]: There is copying of 2 files in 3 directories. I don't think it could have perf troubles with missing layer caching 😁

**Pull request merged**